pages/common/curl.md

```diff
@@ -15,11 +15,15 @@
 
 `curl -O -L -C - {{http://example.com/filename}}`
 
-- Send form-encoded data (POST request of type application/x-www-form-urlencoded):
+- Send form-encoded [d]ata (POST request of type application/x-www-form-urlencoded):
 
 `curl -d {{'name=bob'}} {{http://example.com/form}}`
 
-- Send data, specifying a custom HTTP method, and including an extra header:
+- Send a request with an extra [H]eader:
+
+`curl -H {{'X-My-Header: 123'}} {{http://example.com}}`
+
+- Send [d]ata in JSON format, specifying a custom HTTP method, and including an extra [H]eader:
 
 `curl -d {{'{"name":"bob"}'}} -X {{PUT}} -H {{'Content-Type: application/json'}} {{http://example.com/users/1234}}`
```

Comment:
No, because that would exceed our [maximum number of recommended examples](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines), which is 8.